### PR TITLE
analytics module

### PR DESCRIPTION
**Pull request by annabudziszewska**
## Description
Analytics module

## Motivation and Context
Analytics module: Adobe Analytics and Google Analytics

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the [code styleguide](https://github.com/Cognifide/bobcat/blob/master/CONTRIBUTING.md#styleguide) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: bb-analytics/pom.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  limitations under the License.
+  #L%
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>bobcat</artifactId>
+        <groupId>com.cognifide.qa.bb</groupId>
+        <version>1.4.1-SNAPSHOT</version>
```
Comment (Shaihuludus) [inline bb-analytics/pom.xml]: Version should be updated to 1.5.1-SNAPSHOT. The same for all other dependencies

File: bb-analytics/pom.xml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            <artifactId>bb-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.cognifide.qa.bb</groupId>
+            <artifactId>bb-aem-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.cognifide.qa.bb</groupId>
```
Comment (Shaihuludus) [inline bb-analytics/pom.xml]: Do we need dependency to cucumber module or junit

File: bb-analytics/pom.xml [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    <name>Bobcat Analytics</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.cognifide.qa.bb</groupId>
+            <artifactId>bb-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.cognifide.qa.bb</groupId>
```
Comment (Shaihuludus) [inline bb-analytics/pom.xml]: Why we need aem parts - they will be removed soon and analytics module should be not be restricted only to aem

**Comment by annabudziszewska** [comment]
Closing as deprecated

**Pull request closed without merging**